### PR TITLE
fix: add testing and fix workflows

### PR DESCRIPTION
**Pull request by averikitsch**
This deploys the workflow as a validation of the syntax.

### CODE REVIEW

File: nightly.tests.cloudbuild.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+          fi
+
+          gcloud workflows delete $workflow_name \
+              --location $_REGION \
+              --quiet
+        done
+
+  - id: "Check for Errors"
+    name: "gcr.io/cloud-builders/gcloud:latest"
+    entrypoint: /bin/bash
```
Comment (grayside) [inline nightly.tests.cloudbuild.yaml:33]: Could use an ubuntu or bash container instead for a smaller footprint.

File: nightly.tests.cloudbuild.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+          echo "Deploying $workflow_name with source: $src"
+
+          test="$(gcloud workflows deploy $workflow_name \
+              --source $src \
+              --location $_REGION \
+              --quiet 2>&1)"
+          
+          if [[ $? -ne 0 ]]; then
+              echo "$workflow_name (src: $src) failed!" >> /workspace/failures.txt
+              echo $test >> /workspace/failures.txt
```
Comment (grayside) [inline nightly.tests.cloudbuild.yaml:23]: Could use [tee](https://notes.grayside.dev/posts/tools/gha-env-vars/) to directly output this here and skip the extra cat below. While the difference is small, it reduces the risk of losing the output if something strange happens and the "Check for Errors" step does not execute.

File: nightly.tests.cloudbuild.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    args:
+      - "-c"
+      - |
+        if [ -f /workspace/failures.txt ]; then
+            echo "Tests FAILED!"
+            cat /workspace/failures.txt
+            exit 1
+        else 
+            echo "Tests PASSED!"
+        fi
```
Comment (grayside) [inline nightly.tests.cloudbuild.yaml:43]: Appreciate this aggregation step to clearly signal overall test status!

File: src/connectors/connector_cloudscheduler.workflows.yaml [hunk 4]
```diff
@@ -24,7 +24,7 @@
       - project_id: ${sys.get_env("GOOGLE_CLOUD_PROJECT_ID")}
       - location_id: "us-central1"
       - job_id: "dummy-job-id"
-      - topic_id: "dummy-topic-id"  # Pubsub topic used by Cloud Scheduler as a notification target.
+      - topic: "dummy-topic-id"  # Pubsub topic used by Cloud Scheduler as a notification target.
```
Comment (grayside) [inline src/connectors/connector_cloudscheduler.workflows.yaml:27]: Pre-existing, but this feels like an inclusive language bug.

**Review by grayside — APPROVED** [review]

File: src/connectors/connector_cloudscheduler.workflows.yaml (hunk 4 above)
Comment (averikitsch) [reply src/connectors/connector_cloudscheduler.workflows.yaml:27]: https://github.com/GoogleCloudPlatform/workflows-samples/issues/182

**Pull request merged**